.lando.yml

```diff
@@ -0,0 +1,27 @@
+name: dpul
+services:
+  dpul_development_db:
```

Comment:
Typically we have one database node with two tables. Helps keep our system resources in check.